modules/apps/notification/notification-api/src/main/java/com/liferay/notification/type/BaseNotificationType.java

```diff
@@ -55,13 +57,34 @@
 import java.util.Objects;
 import java.util.Set;
 
+import javax.servlet.http.HttpServletRequest;
+
 import org.osgi.service.component.annotations.Reference;
 
 /**
  * @author Feliphe Marinho
  */
 public abstract class BaseNotificationType implements NotificationType {
 
+	public static void prepareGeneralVariablesTemplate(
```

Comment:
Since this method is used only once in `EmailNotificationType.java`, we can move it there as a private method.